### DISCUSSION TITLE
Abnormal Display of Title Bar After Minimizing Window

### DISCUSSION THREAD

**Discussion by LaiYueTing** [rustdesk/rustdesk#9418] [General]
### Bug Description

When connected to a remote session, the window is maximized, and the title bar is displayed correctly. After minimizing the RustDesk window and reopening it, the buttons in the title bar at the top disappear. The title bar only displays correctly after restoring the window to a windowed state and then maximizing it again.

### How to Reproduce

1. Connect to a remote session
2. Minimize the RustDesk window
3. Maximize the RustDesk window

### Expected Behavior

The title bar displays correctly after maximizing the window.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.2.7 -> 1.3.0 -> 1.3.1

### Screenshots

![image](https://github.com/user-attachments/assets/7336e05c-bea8-43f5-873d-0d33d2c0760b)

**Comment by tomasz1986** [comment]
Same problem here with RustDesk v1.3.1 under Windows 10. It's super annoying 🙁.

Not sure if related, but I've got three displays connected - one 2560x1600 scaled to 250%, one 1920x1200 scaled to 125%, and one 1280x1024 scaled to 125%. However, the first display is "disconnected" in Windows (on purpose), so I'm only using the second and the third one.

**Reply by fufesou** [reply]
Hi, could you please try the latest nightly build?
Not sure if your issue is the same to https://github.com/rustdesk/rustdesk/issues/9479

**Comment by LaiYueTing** [comment]
Hi @rustdesk , can this issue be fixed?

**Comment by fufesou** [comment]
@LaiYueTing Hi, I cannot reproduce this issue.

Could you please: 

1. Try the latest nightly build
2. Give more details about how to reproduce it? A short video or the display settings on the control side

<img width="612" alt="aa" src="https://github.com/user-attachments/assets/a0d9f94a-b79f-4a65-89b2-d57bfe1ce2dc">

**Comment by LaiYueTing** [comment]
Hi @fufesou ,
I tried the latest nightly build, and it's finally back to normal. Thank you so much!
